src/simulation/dynamics/Thrusters/thrusterDynamicEffector/thrusterDynamicEffector.cpp

```diff
@@ -279,6 +279,11 @@ void ThrusterDynamicEffector::computeForceTorque(double integTime, double timeSt
         thrustDirection_B = this->bodyToHubInfo.at(index).dcm_BF * it->thrDir_B;
         thrustLocation_B = this->bodyToHubInfo.at(index).r_FB_B + this->bodyToHubInfo.at(index).dcm_BF * it->thrLoc_B;
 
+        //! - If the connected fuel tank is subject to blow down effects, update them here
```

Comment:
no need for `-` hyphen in the comment

Reply:
Agreed, updated the rest of the comments in the script as well while there.